### ISSUE TITLE
Buttons are pointy in demo apps

### ISSUE THREAD

**Issue by halfnelson** [nativescript-community/ui-material-components#373]
![image](https://user-images.githubusercontent.com/551284/160223647-f22fcbeb-2976-4f21-b401-691b96eb9ba5.png)

### If the demo apps cannot help and there is no issue for your problem, tell us about it

The demo apps (I tried vue and svelte) show the buttons as pointy. These don't look like the material buttons I would expect. This is on a fresh build of this repository, with the style.xml updated to derive from Material instead of Compat.

### Which platform(s) does your issue occur on?
Only tested on android emulator, Pixel 2 API 30

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.2.2
- Cross-platform modules: 8.2.1
- Runtime(s): 8.2.2

**Comment by farfromrefug** [comment]
yes it comes from this https://github.com/nativescript-community/ui-material-components/blob/master/demo-snippets/svelte/install.ts#L16

**Comment by farfromrefug** [comment]
and this https://github.com/nativescript-community/ui-material-components/blob/master/demo-snippets/app.scss#L14
